src/Redux/Book/reducer.js

```diff
@@ -9,7 +9,16 @@ export const deleteBook = (payload) => ({
   type: DELETE_BOOK,
   payload,
 });
-const initialState = [];
+const initialState = [
+  {
+    bookTitle: 'Huckleberry Finn Adventures',
+    bookAuthor: 'Mark Twain',
+  },
+  {
+    bookTitle: 'Tom Swayer Adventures',
+    bookAuthor: 'Mark Twain',
+  },
+];
```

Comment:
- [ ] Whenver I try to remove one of the books from the initial state, both of them are getting removed. After adding new movies they get te deleted correctly, the problem is with only those two, it could be because they do not have an [Video](https://www.loom.com/share/bfe064925ff041fbb7ab5108454a9292)

Reply:
@GzimAsani  Thanks for your review! just updated their Ids and fixed that .